### ISSUE TITLE
Platform Accessories not visible for homebridge-samsung-tizen

### ISSUE THREAD

**Issue by rzwinge** [hoobs-org/HOOBS#549]
**Description**
Platform Accessories not visible for homebridge-samsung-tizen in Hoobs Accessory View.
_I can add the configured TV's over the Apple Home App._

**Expected behavoir**
I can configure a Samsung Tizen Smart TV Platform for multiple TV's in my home.
The config has then multiple devices. You can set multiple key and switch mappings for each device. Each device is his own accessory in homekit.

**Plugin name**
homebridge-samsung-tizen

**Please report the issue to the plugin developer**
I installed a new fresh version of homebridge with config-ui.
The plugin is working great there. It is also shown in the config ui.
So i decided currently to not open a second ticket.

**Version**
3.1.23

**Did you upgrade**
Please let us know if you upgraded from a previous version.
- [ ] Yes
- [x] No

**Pervious version**
-

**Did you orginally upgrade to HOOBS 3 from HOOBS 2.1.1?**
Did you perform an upgrade from HOOBS 2.1.1? Some things are different.
- [ ] Yes
- [x] No

**What device are you using?**
Please let us know the device you are running HOOBS on.
- [ ] HOOBS Box
- [ ] HOOBS SD Card
- [x] Custom Install

**If this is a custom install, what's your operating system?**
I use a Synology NAS with Docker Support. Using the official Hoobs Docker Image.

**Post your config**
```
{
    "server": {
        "port": 4000,
        "origin": "*",
        "autostart": 0,
        "home_setup_id": "X-HM://0023ISYWYD63P",
        "polling_seconds": 5,
        "docker": true
    },
    "client": {
        "default_route": "status",
        "inactive_logoff": 30,
        "theme": "hoobs-dark",
        "locale": "de",
        "temp_units": "celsius",
        "country_code": "DE",
        "postal_code": "00000"
    },
    "bridge": {
        "name": "HOOBS",
        "port": 51826,
        "pin": "031-45-154",
        "username": "00:00:00:00:00:00"
    },
    "description": "",
    "ports": {
        "start": 52000,
        "end": 52999,
        "comment": "Homekit"
    },
    "accessories": [],
    "platforms": [
        {
            "platform": "SamsungTizen",
            "plugin_map": {
                "plugin_name": "homebridge-samsung-tizen"
            },
            "name": "SamsungTizen",
            "devices": [
                {
                    "name": "Wohnzimmer TV",
                    "ip": "192.168.1.100",
                    "mac": "00:00:00:00:00:00"
                }
            ]
        }
    ]
}
```

**Post your log**
```
[4/14/2020, 7:41:05 PM] HOOBS listening on port 4000.
[Homebridge] [4/14/2020, 7:41:05 PM] Loaded config.json with 0 accessories and 1 platforms.
[Homebridge] [4/14/2020, 7:41:06 PM] Loaded plugin: "homebridge-samsung-tizen"
[Homebridge] [4/14/2020, 7:41:06 PM] Registering platform "homebridge-samsung-tizen.SamsungTizen"
[Homebridge] [4/14/2020, 7:41:06 PM] Loading 1 platforms...
[Homebridge] [4/14/2020, 7:41:06 PM] [SamsungTizen] Initializing SamsungTizen platform...
[Homebridge] [4/14/2020, 7:41:06 PM] Loading 0 accessories...
[Homebridge] [4/14/2020, 7:41:06 PM] API launched
[Homebridge] [4/14/2020, 7:41:06 PM] Service is running on port 51826.
[Homebridge] [4/14/2020, 7:41:06 PM] [Wohnzimmer TV] is running on port 52000.
```

**Additional context**
I did some debugging with console.log :P
I checked the HBS call to /accessories, in the response the accessory is missing.
But then i struggled on more debugging in @hoobs/hap. Perhaps the problem is in this package.

Browser Console is empty, no error.

**Comment by mkellsy** [comment]
We are testing 3.1.24 which has an updated HAP-NodeJS. I am hoping to release tommorow.

**Comment by mkellsy** [comment]
I just released 3.1.27
This issue has been fixed.
I recommend updating.

**Comment by BobbySlope** [comment]
closing for now, if issue still persists please reopen and provide more info
